web/embed_style.html

```diff
@@ -28,6 +28,14 @@
     --paper-toggle-button-unchecked-ink-color: var(--text-primary-color);
   }
   
+  .default-text-div {
+    width: 100%;
+    height: 100%;
```

Comment:
`66%`? This would put the watermark 1/3 down from the top; might be nicer then squarely in the middle.